src/objects/camera.lua

```diff
@@ -15,7 +15,7 @@ local camera = require 'camera'
 return {
     init = function(self)
         -- constants
-        self.target_size = self.target_size or 400 -- default target size
+        self.target_size = self.target_size or 600 -- default target size
```

Comment:
In the future I think it will be better to enforce an integer ratio between game pixels and screen pixels. A camera size of 600 stretched to a window 640 pixels wide makes 1 game pixel map to 1.06 screen pixels making things look a little fuzzy.

Reply:
Agreed.

Reply:
I'm working on a new branch which will enforce this ratio and choose a 'scale' to make the camera a sane size. It looks promising -- will open a PR today.